docker/docker-compose.yml

```diff
@@ -12,20 +12,30 @@ services:
   hypertrace-federated-service:
     image: hypertrace/hypertrace-federated-service
     container_name: hypertrace-federated-service
+    volumes: &default-scripts
```

Comment:
```suggestion
   # These scripts are temporary, allowing a temporary override the ENTRYPOINT with a blocking barrier.
   # Unlike service_started condition, these barriers can be longer than a minute. These are temporary
   # because they shadow the ENTRYPOINT, duplicate and ignore the more robust HEALTHCHECK
   # feature available in commands like docker ps.
   #
   # TODO: Remove this after startup improvement fixes
    volumes: &default-scripts
```